packages/astro-font/utils.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="astro/astro-jsx" />
```

Comment:
Are you sure we require it here? It's mostly for Astro files afaik to support the jsx like blocks.

Reply:
This is required to access `astroHTML.JSX.CSSProperties`. Since it looks like we won't support it, I'll remove it

Reply:
Thanks!